test/decorators.py

```diff
@@ -83,6 +83,7 @@ def _wrapper(self, *args, **kwargs):
                     )
                 )
             if not _backend:
+                # pylint: disable=broad-exception-raised
```

Comment:
Perhaps instead change to `IBMBackendError` instead of disabling?
May also be applicable to other exceptions.

Reply:
I don't think `IBMBackendError` would make it more clear since it's a configuration issue so i'll leave it as is for now